site/content/installation/installing-nic/deploy-with-nap-using-helm.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Install NGINX Ingress Controller and NGINX App Protect WAF with Docker and Helm
+toc: true
+weight: 500
+type: how-to
+product: NIC
+docs: DOCS-609
+---
+
+This document describes how to build a local NGINX App Protect WAF Docker image with NGINX Plus Ingress Controller, which can be used to compile WAF policies.
```

Comment:
```suggestion
This document describes how to build a local F5 NGINX App Protect WAF Docker image with NGINX Plus Ingress Controller, which can be used to compile WAF policies.
```

Forgot about this: the very first instance of an NGINX product in documentation must be prefixed with F5 for legal reasons.

Only once, and for the first product - not necessary for everything.